client/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-3.3.0-SNAPSHOT
\ No newline at end of file
+3.2.3-SNAPSHOT
```

Comment:
Please upgrade your openapi generator

Reply:
Adding the specific openapi generator version to the makefile (like in case of other dependencies) might be a good idea.